.circleci/config.yml

```diff
@@ -434,7 +434,7 @@ jobs:
             scripts/run-fsharp-server --published
             # Wait for users to be added so prep.sh works
             ./scripts/devcontainer/_wait-until-apiserver-ready
-            integration-tests/run.sh --concurrency=8 --retry --published
+            integration-tests/run.sh --concurrency=4 --retry --published
```

Comment:
I think this is a good idea. I noticed in CI that this makes each test take much longer (from like 7s to like 15s), and more likely to time out. I hadn't changed it back cause it seemed stable where it was.

Reply:
Yeah I noticed that too - really wonder why that is.